docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -66,3 +67,14 @@ Those types can be used with implicit Index indexer and list patterns.
         if (i is { Length: -1 }) {} // error: impossible under assumption of non-negative length
     }
     ```
+
+5. Starting with Visual Studio 17.1, format specifiers in interpolated strings can not contain curly braces (either `{` or `}`). In previous versions `{{` was interpreted as an escaped `{` and `}}` was interpreted as an escaped `}` char in the format specifier. Now the first `}` char in a format specifier ends the interpolation, and any `{` char is an error.
```

Comment:
@333fred do we specify a VS version?  or a compiler/lang version?

Reply:
AFAIK the old behaviour is a spec violation which we do not want to preserve - so I think the VS version is fine

Reply:
I'm not sure the relation though.  This compiler change will happen regardless of VS version.

Reply:
We specify VS version.